04.ls/ls.rb

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env ruby
+
+dir = Dir.glob '*'
+FILES = dir.count
+COLUMN = 3
+
+if (FILES % COLUMN).zero?
+  LINAGE = FILES / COLUMN
```

Comment:
`LINAGE = FILES / COLUMN` が2箇所に存在しているので、1箇所にまとめられないか検討をお願いします 🙏

Reply:
`FILES / COLUMN `を変数に変更して、if分がスッキリしたので、ワンラインで書くようにしてみました。
```suggestion
ORIGIN_ROW = FILES / COLUMN
ROW = (FILES % COLUMN).zero? ? ORIGIN_ROW : ORIGIN_ROW + 1;
```

Reply:
いいですね 👍 別解として以下のように後置unlessを使う方法もあります。

```ruby
origin_row = FILES / COLUMN
origin_row += 1 unless (FILES % COLUMN).zero?
```

また、Rubyの場合は `Float#ceil` というメソッドがあるのでそれを使っても良いですね。

```ruby
ORIGIN_ROW = (FILES.to_f / COLUMN).ceil
```

さらに、昨年12月に登場したRuby3.2では `Integer#ceildiv` というメソッドが追加されました。これを使うとさらに短く書くことができます 👍 

```ruby
ORIGIN_ROW = FILES.ceildiv(COLUMN)
```

いずれも参考まで 😄

Reply:
条件分岐で実装するという概念で考えていましたが、浮動小数点を繰り上げることで実装できることを理解しました。
物の見方をどの時点から考えるか、大事だなと思いました。
ありがとうございます。
```
ORIGIN_ROW = FILES.ceildiv(COLUMN)
```
こちらで実装しました。